src/displayapp/screens/Music.cpp

```diff
@@ -138,25 +138,29 @@ Music::~Music() {
   lv_obj_clean(lv_scr_act());
 }
 
+// If some of the metadata of a specific track happens to be empty, it should be
+// overwritten, instead of retaining and displaying the older metadata. The artist
+// or the track may not change and get reported again, while some other metadata
+// may not get reported to InfiniTime.
 bool Music::Refresh() {
-  if (artist != musicService.getArtist()) {
+  if (artist != musicService.getArtist() or musicService.getArtist() = "") {
```

Comment:
I guess you meant `or musicService.getArtist() == ""` ?
Or better : `or musicService.getArtist().empty()`

Reply:
oh my god sorry